external/model-preparation-algorithm/configs/segmentation/ocr-lite-hrnet-18-mod2/model.py

```diff
@@ -3,3 +3,11 @@
 ]
 
 load_from = 'https://storage.openvinotoolkit.org/repositories/openvino_training_extensions/models/custom_semantic_segmentation/litehrnet18_imagenet1k_rsc.pth'
+optimizer_config = dict(
+    type='Fp16OptimizerHook', 

```

Comment:
Could you explain why only segmentation task requires whole optimizer_config?

Reply:
mmsegmentation does not handle fp16 configuration. It just build optimizer from optimizer_config. Please check mmsegmentation submodule.

Reply:
Do I need to make it fp16 config and then handle it inside MPA submodule? What do you think? Can you give me some idea?